ServantHDBC/src/Api/Photo.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Api.Photo (PhotoAPI, photoHandlers) where
+
+import Control.Monad.Trans.Except
+import Servant
+
+type PhotoAPI = "photos" :>
+                (Capture "path_segment_1" String :>
```

Comment:
@saurabhnanda 

the effective url is...

/photos/:path_segment_1/:path_segment-2/:geometry_or_style/:original_filename

this is what Capture does...

Query... would mean that every parameter is optional. Capture is part of the URL and mandatory... without that URL is not going to match

if we used Query then it would be...

/photos?path_segment_1=xyz&path_segment_2=abc&geometry_or_style=abc&original_filename=xyz.xyz